schema/config_schema_v3.9.json

```diff
@@ -91,13 +91,42 @@
                 "cache_from": {"$ref": "#/definitions/list_of_strings"},
                 "network": {"type": "string"},
                 "target": {"type": "string"},
-                "shm_size": {"type": ["integer", "string"]}
+                "shm_size": {"type": ["integer", "string"]},
+                "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
+                "isolation": {"type": "string"}
               },
               "additionalProperties": false,
               "patternProperties": {"^x-": {}}
             }
           ]
         },
+        "blkio_config": {
+          "type": "object",
+          "properties": {
+            "device_read_bps": {
+              "type": "array",
+              "items": {"$ref": "#/definitions/blkio_limit"}
+            },
+            "device_read_iops": {
+              "type": "array",
+              "items": {"$ref": "#/definitions/blkio_limit"}
+            },
+            "device_write_bps": {
+              "type": "array",
+              "items": {"$ref": "#/definitions/blkio_limit"}
+            },
+            "device_write_iops": {
+              "type": "array",
+              "items": {"$ref": "#/definitions/blkio_limit"}
```

Comment:
We should consider re-designing this, e.g. to group these options _per device_

```yaml
devices:
    - device: <device>
      iops: <x>
      bps: <x>
```

Reply:
I also think so.
We also should define an `oci` (or `linux` or any comparable name) for cgroup related stuff and raw linux kernel features exposed by the OCI runtime